pygsheets/worksheet.py

```diff
@@ -1710,6 +1710,98 @@ def set_data_validation(self, start=None, end=None, condition_type=None, conditi
             request['setDataValidation']['rule'] = rule
         self.client.sheet.batch_update(self.spreadsheet.id, request)
 
+    @batchable
+    def set_basic_filter(
+        self,
+        start_pos=None,
+        end_pos=None,
+        grange=None,
+        sort_order=None,
+        foreground_color_style=None,
+        background_color_style=None,
+        dimension_index=None,
```

Comment:
a better name is `sort_column`?

Reply:
Sounds good. My major concern was not having 'sort' in the name. as it was not immediately obvious what does this column signifies.

Reply:
I fixed with the commit 374483e